Updated existing connections behavior

ryanbor opened the pull request:
Updated doc to reflect behavior that occurs when the final probe fails in a Load Balancer. Existing connections persist *only* if there is at least one remaining healthy instance. If all instances are unhealthy, the connections *do not* persist and are terminated immediately.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ryanbor committed:
Updated existing connections behavior

Updated doc to reflect behavior that occurs when the final probe fails in a Load Balancer. Existing connections persist *only* if there is at least one remaining healthy instance. If all instances are unhealthy, the connections *do not* persist and are terminated immediately.
